.golangci.yml

```diff
@@ -0,0 +1,26 @@
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - errcheck
+    - goconst
+    - gocyclo
+    - gofmt
+    - goimports
+# TODO: When adding a rule to force comments on exported methods, be sure to uncomment below.
+#    - golint
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - lll
+    - misspell
+    - nakedret
```

Comment:
```suggestion
  enable:
    - bodyclose
    - goconst
    - gocyclo
    - gofmt
    - goimports
# TODO: When adding a rule to force comments on exported methods, be sure to uncomment below.
#    - golint
    - goprintffuncname
    - gosec
    - interfacer
    - lll
    - misspell
    - nakedret
```

How about clearing the `Enabled By Default Linters` item?
https://golangci-lint.run/usage/linters/#enabled-by-default-linters

Reply:
I chose this style([disable-all: true](https://github.com/yorkie-team/yorkie/pull/106/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R2)) because it doesn't seem explicit that there is a default linter. Is it a little too much?
https://github.com/yorkie-team/yorkie/pull/106/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R2

Reply:
Ah! Did not recognize `disable-all: true`. If so, it looks like no problem! :)

Reply:
@hackerwins 
Can you explain why you selected only some of the linters that are enabled by default?

Reply:
I added the strict linters suggested in [a post](https://medium.com/wesionary-team/introduction-to-linters-in-go-and-know-about-golangci-lint-6486fb2864b3). I didn't notice that the default linters is missing. 

I think it would be better to allow the default linters as new suggestions may be added to the default.

Reply:
That's a good opinion. I think we can check the code more strictly using the default linters. After using it, if there are any inconveniences to us in the future, I think we can select certain linters at that time.